services/src/main/java/io/scalecube/services/ServiceRegistryImpl.java

```diff
@@ -115,7 +119,7 @@ private void loadMemberServices(DiscoveryType type, Member member) {
   /**
    * register a service instance at the cluster.
    */
-  public void registerService(Object serviceObject) {
+  public void registerService(Object serviceObject, Tag[] tags) {
```

Comment:
i will explain you can see that everywhere its Tag except when reading and writing it to metadata.